src/test/groovy/com/netgrif/application/engine/importer/ImporterTestGroovy.groovy

```diff
@@ -1,129 +1,129 @@
-        assert testCase.dataSet.get(TEXT_FIELD).behavior.get("1") == [FieldBehavior.HIDDEN] as Set<FieldBehavior>
-        assert testCase.dataSet.get(ENUMERATION_FIELD).behavior.get("1") == [FieldBehavior.VISIBLE] as Set<FieldBehavior>
-        assert testCase.dataSet.get(ENUMERATION_MAP_FIELD).behavior.get("1") == [FieldBehavior.EDITABLE] as Set<FieldBehavior>
-        assert testCase.dataSet.get(MULTICHOICE_FIELD).behavior.get("1") == [FieldBehavior.REQUIRED] as Set<FieldBehavior>
-        assert testCase.dataSet.get(MULTICHOICE_MAP_FIELD).behavior.get("1") == [FieldBehavior.IMMEDIATE] as Set<FieldBehavior>
-        assert testCase.dataSet.get(BOOLEAN_FIELD).behavior.get("1") == [FieldBehavior.OPTIONAL] as Set<FieldBehavior>
-        assert testCase.dataSet.get(DATE_FIELD).behavior.get("1") == [FieldBehavior.EDITABLE, FieldBehavior.REQUIRED] as Set<FieldBehavior>
-        assert testCase.dataSet.get(DATETIME_FIELD).behavior.get("1") == [FieldBehavior.IMMEDIATE, FieldBehavior.REQUIRED] as Set<FieldBehavior>
-        assert testCase.dataSet.get(FILE_FIELD).behavior.get("1") == [FieldBehavior.IMMEDIATE, FieldBehavior.FORBIDDEN] as Set<FieldBehavior>
-        assert testCase.dataSet.get(FILE_LIST_FIELD).behavior.get("1") == [FieldBehavior.HIDDEN, FieldBehavior.OPTIONAL] as Set<FieldBehavior>
-        assert testCase.dataSet.get(USER_FIELD).behavior.get("1") == [FieldBehavior.HIDDEN, FieldBehavior.IMMEDIATE] as Set<FieldBehavior>
-        assert testCase.dataSet.get(BUTTON_FIELD).behavior.get("1") == [FieldBehavior.EDITABLE, FieldBehavior.REQUIRED, FieldBehavior.IMMEDIATE] as Set<FieldBehavior>
-    }
-
-    @Test
-    void enumerationMultichoiceOptionsTest() throws IOException, MissingPetriNetMetaDataException {
-        PetriNet net = petriNetService.importPetriNet(new ClassPathResource("/enumeration_multichoice_options.xml").getInputStream(), VersionType.MAJOR, superCreator.getLoggedSuper()).getNet()
-
-        assert net != null
-
-        ChoiceField multichoice = (ChoiceField) net.getDataSet().get(MULTICHOICE_FIELD)
-        ChoiceField multichoice_like_map = (ChoiceField) net.getDataSet().get(MULTICHOICE_LIKE_MAP_FIELD)
-        ChoiceField enumeration = (ChoiceField) net.getDataSet().get(ENUMERATION_FIELD)
-        ChoiceField enumeration_like_map = (ChoiceField) net.getDataSet().get(ENUMERATION_LIKE_MAP_FIELD)
-
-        assert multichoice.getChoices() == multichoice_like_map.getChoices()
-        assert enumeration.getChoices() == enumeration_like_map.getChoices()
-
-        assert multichoice.getValue() == multichoice_like_map.getValue()
-        assert enumeration.getValue() == enumeration_like_map.getValue()
-
-        assert multichoice.getDefaultValue() == multichoice_like_map.getDefaultValue()
-        assert enumeration.getDefaultValue() == enumeration_like_map.getDefaultValue()
-    }
-}
+//package com.netgrif.application.engine.importer
```

Comment:
this file should be outright removed. If necessary it can still be found in git history